lib/mpl_toolkits/tests/test_mplot3d.py

```diff
@@ -20,16 +20,37 @@ def test_bar3d():
         ax.bar(xs, ys, zs=z, zdir='y', color=cs, alpha=0.8)
 
 
-@cleanup
-def test_bar3d_dflt_smoke():
+@image_comparison(
+    baseline_images=['bar3d_shaded'],
+    remove_text=True,
+    extensions=['png']
+)
+def test_bar3d_dflt_actually3d():
```

Comment:
I prefer test names and baseline image names that are the same or very nearly the same. It makes it easy to match a bad image file with the test in the code.